x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/data_view/static.spec.ts

```diff
@@ -0,0 +1,283 @@
+import type { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
+import {
+  SupertestReturnType,
+  ApmApiError,
+} from '../../../../../../apm_api_integration/common/apm_api_supertest';
+
+export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const apmApiClient = getService('apmApi');
+  const roleScopedSupertest = getService('roleScopedSupertest');
+  const synthtraceService = getService('synthtrace');
+  const logger = getService('log');
+  const dataViewPattern =
+    'traces-apm*,apm-*,traces-*.otel-*,logs-apm*,apm-*,logs-*.otel-*,metrics-apm*,apm-*,metrics-*.otel-*';
+
+  function createDataViewWithWriteUser({ spaceId }: { spaceId: string }) {
+    return apmApiClient.writeUser({
+      endpoint: 'POST /internal/apm/data_view/static',
+      spaceId,
+    });
+  }
+
+  function createDataViewWithReadUser({ spaceId }: { spaceId: string }) {
+    return apmApiClient.readUser({
+      endpoint: 'POST /internal/apm/data_view/static',
+      spaceId,
+    });
+  }
+
+  describe('Data view static', () => {
+    let supertest: SupertestWithRoleScope;
+
+    before(async () => {
+      supertest = await roleScopedSupertest.getSupertestWithRoleScope('editor', {
+        withInternalHeaders: true,
+      });
+    });
+
+    after(async () => {
+      await supertest.destroy();
+    });
+
+    function deleteDataView(spaceId: string) {
+      return supertest.delete(
+        `/s/${spaceId}/api/saved_objects/index-pattern/${getStaticDataViewId(spaceId)}?force=true`
+      );
+    }
+
+    function getDataView({ spaceId }: { spaceId: string }) {
+      const spacePrefix = spaceId !== 'default' ? `/s/${spaceId}` : '';
+      return supertest.get(
+        `${spacePrefix}/api/saved_objects/index-pattern/${getStaticDataViewId(spaceId)}`
+      );
+    }
+
+    function getDataViewSuggestions(field: string) {
+      return supertest
+        .post(`/internal/kibana/suggestions/values/${dataViewPattern}`)
+        .set(ELASTIC_HTTP_VERSION_HEADER, '1')
+        .send({ query: '', field, method: 'terms_agg' });
+    }
+
+    describe('no mappings exist', () => {
+      let response: SupertestReturnType<'POST /internal/apm/data_view/static'>;
+      before(async () => {
+        response = await createDataViewWithWriteUser({ spaceId: 'default' });
+      });
+
+      it('does not create data view', async () => {
+        expect(response.status).to.be(200);
+        expect(response.body).to.eql({
+          created: false,
+          reason: 'No APM data',
+        });
+      });
+
+      it('cannot fetch data view', async () => {
+        const res = await getDataView({ spaceId: 'default' });
+        expect(res.status).to.be(404);
+        expect(res.body.message).to.eql(
+          'Saved object [index-pattern/apm_static_data_view_id_default] not found'
+        );
+      });
+    });
+
+    // FLAKY: https://github.com/elastic/kibana/issues/177120
```

Comment:
can you please clean up the comment? Also can you please run the flaky runner to verify it's not flaky anymore

Reply:
checked and they are working fine, removed comment!